ansible-chatbot-deploy.yaml

```diff
@@ -44,18 +38,18 @@ spec:
       labels:
         app.kubernetes.io/name: ansible-chatbot
     spec:
+      imagePullSecrets:
```

Comment:
This may not be strictly necessary; but it is a good place-holder/reminder.

Reply:
+1